vmdk_plugin/drivers/photon/photon_driver.go

```diff
@@ -586,6 +592,14 @@ func (d *VolumeDriver) Unmount(r volume.UnmountRequest) volume.Response {
 		).Error("Refcount error - still trying to unmount...")
 	}
 
+	// defensive approach to not unmount volumes until refcounting has been successful
+	if d.refCounts.GetSuccess() != true {
+		log.WithFields(
+			log.Fields{"name": r.Name},
+		).Info("Reference count map is not initialized. Unmount is delayed")
+		return volume.Response{Err: "Refcount Error"}
```

Comment:
Why do we want the unmount 'refcount error' in this case exposed to the docker command ? I think it will only confuse the upstairs. we need to pretend all is good and return success